app/1.0/docs/devguide/quick-tour.md

```diff
@@ -156,7 +156,7 @@ can optionally support two-way binding.
 This example uses two-way binding: binding the value of a custom input element (`iron-input`)
 to the element's `owner` property, so it's updated as the user types.
 
-<demo-tabs selected="0" src="http://plnkr.co/edit/OXaNcCl7qkeMwXEtiTxX?p=preview">
+<demo-tabs selected="0" src="http://plnkr.co/edit/DIA7weGNRf9Ejxp6xyh1?p=preview">
```

Comment:
"This is a Daniel's editable-name-tag." Should be "This is Daniel's editable-name-tag."

Or "This is-a Mario's editable-name-tag if you prefer."